dbt/adapters/base/impl.py

```diff
@@ -764,7 +763,9 @@ def get_missing_columns(
         return [col for (col_name, col) in from_columns.items() if col_name in missing_columns]
 
     @available.parse_none
-    def valid_snapshot_target(self, relation: BaseRelation) -> None:
+    def valid_snapshot_target(
```

Comment:
Anything that calls this method with the new kwarg will need to bump its lower bound pin on `dbt-adapters`. That's fine, just making a note of it.

Reply:
So the lower bound pin of dbt-spark and dbt-postgres can be updated in the appropriate pull requests. What about the adapters without a pr?  Like dbt-snowflake and dbt-redshift?  Their code doesn't call the new macro directly, but will call it via the dbt-adapters code.